src/ol_infrastructure/lib/aws/rds_helper.py

```diff
@@ -41,4 +41,4 @@ def parameter_group_family(engine: Text, engine_version: Text) -> Text:
     :rtype: Text
     """
     engine_details = rds_client.describe_db_engine_versions(Engine=engine, EngineVersion=engine_version)
-    return engine_details['DBParameterGroupFamily']
+    return engine_details['DBEngineVersions'][0]['DBParameterGroupFamily']
```

Comment:
Why are we only interested in the first item?

Reply:
Because the combination of engine name and version should only result in one result.

Reply:
Even when u have multiple dbs with the same engine and version?

Reply:
Nevermind